docker/unified/build-image-rootless.sh

```
@@ -0,0 +1,102 @@
+NO_CACHE=false
+
+for arg in "$@"; do
+    case $arg in
+        --cuda)
+            BACKEND="cuda"
+            ;;
+        --vulkan)
+            BACKEND="vulkan"
+            ;;
+        --no-cache)
+            NO_CACHE=true
```

Shellcheck (0.11.0)

[warning] NO_CACHE appears unused. Verify use (or export if used externally).

(SC2034)